pkg/controller/wavefrontproxy/wavefrontproxy_controller.go

```diff
@@ -123,7 +124,11 @@ func (r *ReconcileWavefrontProxy) Reconcile(request reconcile.Request) (reconcil
 // If exists, then brings it from current state -> desired state.
 func (r *ReconcileWavefrontProxy) reconcileProxy(ip *InternalWavefrontProxy, reqLogger logr.Logger) (reconcile.Result, error) {
 	desiredDep := newDeployment(ip)
-
+	if ip.instance.Spec.Openshift && ip.instance.Spec.StorageClaimName != "" {
```

Comment:
We need this only when creating a new Proxy ?

Reply:
Need this when creating new proxy.  Done moved to 140.